lib/contract/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+/**
+ * @module nahmii-sdk
+ */
+
+const ethers = require('ethers');
+const contractAbstractions = require('./abstractions');
+
+/**
+ * @class NahmiiContract
+ * A class providing access to the various nahmii contracts by name.
+ * @alias module:nahmii-sdk
+ * @example
+ * const {NahmiiContract} = require('nahmii-sdk');
+ * ...
+ * const niiContract = new NahmiiContract('NahmiiToken', nahmiiProvider);
+ * const balance = await niiContract.balanceOf(someWalletAddress);
+ */
+class NahmiiContract extends ethers.Contract {
```

Comment:
Nice, I think this can replace some code in Trade API

Reply:
Just give it a provider (or fake one) and it should work fine.